xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-test/xwiki-platform-administration-test-docker/src/test/it/org/xwiki/administration/test/ui/AdministrationIT.java

```diff
@@ -71,11 +72,11 @@ void verifyAdministrationSections(TestUtils setup)
         assertTrue(wikiAdministrationPage.hasNotSection("PageRights"));
 
         // Select XWiki page administration.
-        setup.gotoPage("NonExistentSpace", "WebHome");
+        setup.gotoPage(testReference);
         page = new AdministrablePage();
         AdministrationPage pageAdministrationPage = page.clickAdministerPage();
 
-        assertEquals("Page Administration: NonExistentSpace", pageAdministrationPage.getDocumentTitle());
+        assertEquals("Page Administration: AdministrationIT.verifyAdministrationSections", pageAdministrationPage.getDocumentTitle());
```

Comment:
This won't work if you execute the whole test suite as the test class will be NestedAdministrationIT. You need to use a serialization of the testReference.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3603/commits/61c26755c85b3a8b077e4eff4caea25e9799360a . I tested with the singular test already, but I'm currently running the whole test suite. I'll update this if the whole test suite does not pass somehow.